proto/shared_constants.proto

```diff
@@ -159,6 +163,26 @@ message PortType {
   }
 }
 
+/*
+ * TestResult
+ * TODO: Replace this enum with SessionResult in DAQ.
+ */
+
+/*
+ * Session result codes.
+ */
+message SessionResult {
+  SessionResult.ResultCode code = 1;
```

Comment:
If the message contains a data field, then I don't think it would be a good idea to define it in shared_constants.proto...

Reply:
is the field actually used?  Seems like for this just need the constants?

Reply:
I copied this directly from your PR. I didn't know what it was intended so I left it like that.